denizen_scripts/survival/repo_link/spawn_scripts/survival_rtp.dsc

```diff
@@ -27,7 +27,20 @@ survival_falloff_rtp:
   debug: false
   events:
     after player enters spawn_below:
+    - if !<player.flag[RecentRTP]>:
```

Comment:
this should be `player.has_flag` to determine if the player has the flag, or not
```suggestion
    - if !<player.has_flag[RecentRTP]>:
```